### PR TITLE
Fix code background color light mode [Fixes #52]

### LINKED ISSUE #52: [DESGIN BUG] Change the background color for the code snippets
Screen Shot 2022-11-25 05 34 06 PM

### PR DESCRIPTION
**Pull request by ai-slave**
Fixes #52 , #50

### CODE REVIEW

File: src/components/UI/downloads/SpecificVersionsSection.tsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ export const SpecificVersionsSection: FC<Props> = ({ children }) => {
         borderRight={{ base: 'none', md: '2px' }}
         borderColor='primary !important'
         flex='none'
+        width={{ base: 'auto', md: '300px' }}
```
Comment (wackerow) [inline src/components/UI/downloads/SpecificVersionsSection.tsx:25]: I think this has been fixed with #64

Comment (wackerow) [reply src/components/UI/downloads/SpecificVersionsSection.tsx:25]:
```suggestion
```

**Review by wackerow — CHANGES_REQUESTED** [review]
If we remove the `width` addition, the other change lgtm, thanks @ai-slave!

**Review by wackerow — APPROVED** [review]

**Comment by nhsz** [comment]
Closing as this was already fixed

**Comment by wackerow** [comment]
Agree.. @nhsz issue #52 has not been fixed yet, and this PR would fix it:


That issue requests that the code background be switched to use `gray.100` instead of `gray.200`.. Going to re-open this.

**Comment by wackerow** [comment]
Sorry, my bad. Got these flipped—source of truth says to use `gray.200` which is what master shows. Closing back out

**Pull request closed without merging**